src/pkgcheck/checks/codingstyle.py

```diff
@@ -79,6 +79,29 @@ def feed(self, pkg):
                     yield DeprecatedEapiCommand(name, line=call, lineno=lineno+1, eapi=pkg.eapi, pkg=pkg)
 
 
+class EendMissingArg(results.LineResult, results.Style):
+    """Eend is called without an argument"""
+
+    @property
+    def desc(self):
+        return f'Call to eend is missing an argument, on line {self.lineno}'
```

Comment:
Reformat this similar to how I showed in the previous PR.

Reply:
Done.